### ISSUE TITLE
Feature idea - Screenshots of failed scenarios

### ISSUE THREAD

**Issue by zacoid55** [lyndsey-ferguson/fastlane-plugin-test_center#11]
I'm wondering whether there's a way to extract the last screenshot that gets saved on a failed test and add it to the report. I think this happens in other reporting tools but don't quote me on that.

Slap me if you think this might be a silly idea!

**Comment by lyndsey-ferguson** [comment]
No, it is definitely a good idea. fastlane already has this for `snapshot`: it does require the Xcode developer to add a rotation to the "unknown" orientation to work.
